test/fixtures/v2/pdt_lamp_unwrapped.json

```diff
@@ -0,0 +1,321 @@
+                        },
+                        {
+                            "id": "NRIC-FIN",
+                            "value": "S9098989Z"
+                        }
+                    ],
+                    "name": [
+                        {
+                            "text": "Tan Chen Chen"
+                        }
+                    ],
+                    "gender": "female",
+                    "birthDate": "1990-01-15"
+                }
+            },
+            {
+                "fullUrl": "urn:uuid:7729970e-ab26-469f-b3e5-36a42ec24146",
+                "resource": {
+                    "resourceType": "Observation",
+                    "specimen": {
+                        "type": "Specimen",
+                        "reference": "urn:uuid:0275bfaf-48fb-44e0-80cd-9c504f80e6ae"
+                    },
+                    "performer": [
+                        {
+                            "type": "Practitioner",
+                            "reference": "urn:uuid:3dbff0de-d4a4-4e1d-98bf-af7428b8a04b"
+                        },
+                        {
+                            "id": "LHP",
+                            "type": "Organization",
+                            "reference": "urn:uuid:fa2328af-4882-4eaa-8c28-66dab46950f1"
+                        },
+                        {
+                            "id": "AL",
+                            "type": "Organization",
+                            "reference": "urn:uuid:839a7c54-6b40-41cb-b10d-9295d7e75f77"
+                        }
+                    ],
+                    "identifier": [
+                        {
+                            "id": "ACSN",
+                            "type": {
+                                "coding": [
+                                    {
+                                        "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                                        "code": "ACSN",
+                                        "display": "Accession ID"
+                                    }
+                                ]
+                            },
+                            "value": "123456789"
+                        }
+                    ],
+                    "category": [
+                        {
+                            "coding": [
+                                {
+                                    "system": "http://snomed.info/sct",
+                                    "code": "840539006",
+                                    "display": "COVID-19"
+                                }
+                            ]
+                        }
+                    ],
+                    "code": {
+                        "coding": [
+                            {
+                                "system": "http://loinc.org",
+                                "code": "96986-5",
+                                "display": "Lucira Test"
+                            }
```

Comment:
Can we use the full LOINC display name here?

```text
SARS-CoV-2 (COVID-19) N gene [Presence] in Nose by NAA with non-probe detection
```

Reference: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/wiki/4.-Recommended-Coding-Set#type-of-test-loinc